src/main/java/ch/njol/skript/conditions/CondElytraBoostConsume.java

```diff
@@ -0,0 +1,58 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import com.destroystokyo.paper.event.player.PlayerElytraBoostEvent;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Will Consume Boosting Firework")
+@Description("Checks to see if the firework used in an 'elytra boost' event will be consumed.")
+@Examples({
+	"on elytra boost:",
+		"\tif the used firework will be consumed:",
+			"\t\tprevent the used firework from being consume"
+})
+@RequiredPlugins("Paper")
+@Since("INSERT VERSION")
+public class CondElytraBoostConsume extends Condition {
+
+	private static final boolean ELYTRA_BOOST_EXISTS = Skript.classExists("com.destroystokyo.paper.event.player.PlayerElytraBoostEvent");
```

Comment:
Since this is only used in the static field, you can just check this condition there